kafka/integration_test.go

```diff
@@ -2684,6 +2685,115 @@ func (its *IntegrationTestSuite) TestAdminClient_UserScramCredentials() {
 	}
 }
 
+// Tests ListOffsets API which describes
+// the offset of a TopicPartiton corresponding to the OffsetSpec provided.
+func (its *IntegrationTestSuite) TestListOffsets() {
+	t := its.T()
+	bootstrapServers := testconf.Brokers
+
+	// Create a new AdminClient.
+	// AdminClient can also be instantiated using an existing
+	// Producer or Consumer instance, see NewAdminClientFromProducer and
+	// NewAdminClientFromConsumer.
+	a, err := NewAdminClient(&ConfigMap{"bootstrap.servers": bootstrapServers})
+	if err != nil {
+		t.Fatalf("Unable to create Admin Client, error : %s", err)
+	}
+
+	// Contexts are used to abort or limit the amount of time
+	// the Admin call blocks waiting for a result.
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	requests := make(map[TopicPartition]int64)
+	goTopic := "topicname"
+
+	var topics []TopicSpecification
+	topics = append(topics, TopicSpecification{Topic: goTopic, NumPartitions: 1, ReplicationFactor: 1})
+	create_topic_result, create_topic_err := a.CreateTopics(ctx, topics)
+	if create_topic_err != nil {
+		t.Fatalf("Unable to create topic, error : %s", err)
```

Comment:
Now that assertions are available, use them.
```
assert := its.Assert()
....
....
assert.Nil(err, "CreateTopics should succeed")
```
and other methods on assert.
Everywhere, it's better readability and we will use this library going forward